tests/admin/conftest.py

```diff
@@ -86,3 +88,46 @@ def expired_security_code(app):
 
         db.session.delete(security_code_)
         db.session.commit()
+
+
+@pytest.fixture()
+def submitted_application(app):
+    with app.app_context():
+        with app.test_request_context():
+            application = create_test_applications(status=ApplicationStatus.SUBMITTED, number_of_applications=1)
```

Comment:
Not quite. I wouldn't use this helper function. I would create multiple Application records similarly to how it's done in create_test_applications.